portia/portia.py

```diff
@@ -24,6 +42,15 @@ def __init__(self, redis, prefix="portia:", network_prefix_mapping=None):
         self.redis = redis
         self.prefix = prefix
         self.network_prefix_mapping = network_prefix_mapping or {}
+        self.timezone = UTC()
+
+    def to_utc(self, timestamp):
+        if timestamp.tzinfo:
+            return timestamp.astimezone(self.timezone)
+        return timestamp.replace(tzinfo=self.timezone)
+
+    def now(self):
+        return self.to_utc(datetime.now())
```

Comment:
This will return local time as if it were UTC, which is only correct when local time has zero UTC offset. You probably want `datetime.utcnow()` here.